src/io/flutter/inspector/FlutterWidget.java

```diff
@@ -26,11 +28,64 @@
  */
 public class FlutterWidget {
 
+  enum Category {
+    // TODO(pq): fill in undefined widgets.
+    ACCESSIBILITY("Accessibility", null), ANIMATION_AND_MOTION("Animation and Motion", null),
```

Comment:
Not: is this formatted? I'm a bit surprised there are multiple enum entries per line.

Reply:
Sort of. 😉